src/partials/header.html

```diff
@@ -0,0 +1,35 @@
+<header class="header__line">
+    <a class="logo" href="index.html" title="Main page">
+        <svg class="logo__icon" width="35.28" height="42.8">
+            <use href="#"></use>
+          </svg>
+        <svg class="logo__icon" width="40.96" height="42.8">
+            <use href="#"></use>
+        </svg>
+        <svg class="logo__icon" width="40.96" height="42.8">
+            <use href="#"></use>
+        </svg>      
+    </a>
+    <ul class="menu__list">
+        <li class="menu__item">
+            <a class="menu__link" href="">Anemone</a>
+        </li>
+        <li class="menu__item">
+            <a class="menu__link" href="#">About us</a>
+        </li>
+        <li class="menu__item">
+            <a class="menu__link" href="#">Store</a>
+        </li>
+        <li class="menu__item">
+            <a class="menu__link" href="#">Delivery</a>
+        </li>
+        <li class="menu__item">
+            <a class="menu__link" href="#">Blog</a>
+        </li>
+    </ul>
+    <a href="tel:+48111111111" class="contact__link">
+        <svg class="contact__icon" width="22" height="22">
+            <use href="#"></use>
+        </svg>
+    </a>       
```

Comment:
jak najbardziej git tylko tą ikone potraktuj jako button bo to będzie nam otwierać modal i brakuje jeszcze ikony hamburgera gdzie też będzie to jako button bo będzie nam otwierać menu mobilne